static/templates/user_info_popover_content.handlebars

```diff
@@ -27,8 +27,11 @@
         {{#if user_time}}
         <li>{{ user_time }} {{#tr this}}Local time{{/tr}}</li>
         {{/if}}
-
+        {{#if_and is_bot bot_owner }}
+        <li>{{ user_type }} {{#tr this}}owned by{{/tr}} <a class="bot-info-popover" data-bot-owner-id='{{ bot_owner.user_id }}'> {{bot_owner.full_name}}</a></li>
```

Comment:
Any idea how to handle this? I tried to change it to
```
<li>{{#tr this}}__user_type__ owned by{{/tr}}  <a class="bot-info-popover" data-bot-owner-id='{{ bot_owner.user_id }}'> {{bot_owner.full_name}}</a></li>
```
but here `user_type` is already translated, `i.18n.t('Bot')`.

Reply:
That's roughly what you wanted to do (main other issue is that the translation string shouldn't end at `by`, since some other languages have a different language order), it should end at the `</a>` tag.

There's no double-translation problem, because the `__user_type__` syntax inserts `user_type` as a variable.

Reply:
(Though actually since `user_type` is always `Bot` for this string, we should probably just put that in the string here in any case)